lib/i18n_digests.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class I18nDigests
+  class << self
+    def build_digests(available_locales)
+      available_locales.each do |locale|
+        (self.i18n_digests ||= {})[locale.to_sym] = locale_file_digest(locale)
```

Comment:
I thought that we are missing the setter `i18n_digests=` but then I found that the application config provides hashes for any key already. This code would fail if `i18n_digests` would return `nil`. So we can omit the backup value as well:
```suggestion
        self.i18n_digests[locale.to_sym] = locale_file_digest(locale)
```

Reply:
Yeah, the `x` namespace under `Rails.application.config` returns a generic and flexible configuration object that's pre-initialized and can be used to set any custom config options :+1:

I think you're right that we don't need the `||= {}` here :+1: